### PR TITLE
Add Monitoring Configuration

### PR DESCRIPTION
**Pull request by trthhrtz**

### CODE REVIEW

File: hydrosdk/modelversion.py [hunk 1]
```diff
@@ -504,7 +506,7 @@ def __init__(self, cluster: Cluster, id: int, model_id: int, name: str, version:
                  contract: ModelContract, status: Optional[ModelVersionStatus], image: Optional[DockerImage], 
                  runtime: Optional[DockerImage], is_external: bool, 
                  metadata: Optional[Dict[str, str]] = None, install_command: Optional[str] = None, 
-                 training_data: Optional[str] = None):
+                 training_data: Optional[str] = None, monitoring_configuration: dict = None):
```
Comment (KineticCookie) [inline hydrosdk/modelversion.py:509]:
```suggestion
                 training_data: Optional[str] = None, monitoring_configuration: Optional[MonitoringConfiguration] = None):
```

**Review by KineticCookie — CHANGES_REQUESTED** [review]
Define monitoring configuration as a class

File: hydrosdk/modelversion.py (hunk 1 above)
Comment (trthhrtz) [reply hydrosdk/modelversion.py]: Fixed

**Comment by trthhrtz** [comment]
MonitoringConfiguration class is defined

**Review by KineticCookie — APPROVED** [review]

**Pull request merged**